add authorization to all references missing it

techsmyth opened the pull request:
<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Chores**
	- Updated database to ensure all references have associated authorization policies. No user-facing changes.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

techsmyth committed:
add authorization to all references missing it

bobbykolev reviewed (dismissed):
Looks good to me, thanks Neil. I missed that one when adding the contributors' social references.

valentinyanakiev committed:
Merge branch 'develop' into server-5166

valentinyanakiev committed:
Update src/migrations/1748628720167-refAuths.ts

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

valentinyanakiev committed:
Uses parameterized queries in migration

Updates the database migration to use parameterized
queries instead of string interpolation to prevent
SQL injection vulnerabilities.
